src/ucp/core/ucp_vfs.inl

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCP_VFS_INL_
+#define UCP_VFS_INL_
+
+#include <ucs/datastruct/string_buffer.h>
+#include <ucs/sys/compiler_def.h>
+#include <ucs/sys/sock.h>
+
+static UCS_F_ALWAYS_INLINE void
+ucp_vfs_read_ip(struct sockaddr* sockaddr, ucs_string_buffer_t *strb)
+{
```

Comment:
why does it need to be inline functions?
maybe move all vfs stuff to ucp_vfs.c?